client/js/screens/Locker/Locker.js

```diff
@@ -1,10 +1,109 @@
-import React from "react";
-import { Text, View } from "react-native";
-
-const Locker = ({ params }) => (
-  <View>
-    <Text>This is the Locker Screen</Text>
-  </View>
-);
+import React, { Component } from "react";
+import {
+  Text,
+  View,
+  TouchableOpacity,
+  Image,
+  Linking,
+  Platform,
+  Modal,
+  ScrollView
+} from "react-native";
+import styles from "./styles";
+import PhotoCarousel from "../../components/PhotoCarousel";
+import { Gravatar } from "react-native-gravatar";
+
+class Locker extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalVisible: true
+    };
+  }
+  setModalVisible(visible) {
+    this.setState({ modalVisible: visible });
+  }
+  render() {
+    const { lockerinfo } = this.props;
+
+    const srcLatitude = 49.2633479;
+    const srcLongitude = -123.140316;
```

Comment:
This can be read from the device API instead of being hardcoded.

Reply:
They're dummy data for now while I wait for Jeremiah's map issues to be merged.